tokio-threadpool/src/pool/mod.rs

```diff
@@ -60,6 +61,9 @@ pub(crate) struct Pool {
     // This will *usually* be a small number.
     pub workers: Box<[worker::Entry]>,
 
+    // External queue of tasks
```

Comment:
Nit: I don't find this comment particularly helpful --- the "external queue" part just repeats the name of the field, and the "of tasks" part is already pretty clearly encoded in the type signature. Perhaps this comment ought to explain what the queue is _for_?